.github/ISSUE_TEMPLATE/00_BB_issue_template.yml

```diff
@@ -0,0 +1,59 @@
+name: BB bug
+description: File an issue for a BB.
+title: "[BB Bug]:"
+labels: [BB, bug]
+
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thanks for taking the time to fill out this form!
+
+- type: input
+  id: contact
+  attributes:
+    label: Contact Details
+    description: How can we get in touch with you if we need more info?
+    placeholder: eg. email@example.com
+  validations:
+    required: false
+
+- type: dropdown
+  id: layer
+  attributes:
+    label: Layer
+    description: Please select the corresponding Layer of the affected BB
+    multiple: true
+    options:
+    - AppLayer
+    - MWLayer
+    - OSLayer
+    - HWLayer
+  validations:
+    required: true
+
+- type: input
+  id: fc
```

Comment:
If I'm already identifying a specific BB by name/id (below), why do I need to specify functional cluster in addition? Isn't this redundant?

Reply:
We agree that this could be a potantial source of inconstistency but it makes identifying and finding the affected BB much more straight forward.